lightblue/src/main/java/org/esbtools/eventhandler/lightblue/client/BulkLightblueRequester.java

```diff
@@ -85,7 +85,7 @@ public BulkLightblueRequester(LightblueClient lightblue) {
     }
 
     @Override
-    public TransformableFuture<LightblueDataResponses> request(AbstractLightblueDataRequest... requests) {
+    public TransformableFuture<LightblueDataResponses> request(CRUDRequest... requests) {
```

Comment:
I am looking to see if I can use LightblueDataRequest anywhere, but CRUDRequest has the getOperation method which is required in DataBulkRequest.